go.mod

```diff
@@ -67,6 +67,8 @@ require (
 	google.golang.org/genproto/googleapis/api v0.0.0-20241202173237-19429a94021a
 )
 
+require github.com/gogo/status v1.1.0 // indirect
```

Comment:
weird that it got split :thinking:

Reply:
yup bit weird, but this is vanilla result of `go mod tidy` 🤔